worlds/kh1/Options.py

```diff
@@ -213,11 +239,17 @@ class ReportsInPool(Range):
     range_start = 0
     range_end = 13
 
-class RandomizeKeybladeStats(DefaultOnToggle):
+class KeybladeStats(Choice):
```

Comment:
You may want to mark the old option as Removed. This will make it so old yamls throw errors instead of just using the default value. There's [an example](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/alttp/Options.py#L825) in ALttP